Cesium.CodeGen/Extensions/TypeSystemEx.cs

```diff
@@ -16,13 +17,30 @@ public static class TypeSystemEx
         var typeName = components[0];
         var methodName = components[1];
 
-        // TODO: Lookup in referenced assemblies.
-        var runtimeAssembly = typeof(Console).Assembly;
-        var type = runtimeAssembly.GetType(typeName);
-        if (type == null) return null;
-        var method = type.GetMethod(methodName);
+        var method = FindType(context.ImportAssemblies, typeName, methodName);
         if (method == null) return null;
 
         return context.Module.ImportReference(method);
     }
+
+    private static MethodInfo? FindType(Assembly[] assemblies, string typeName, string methodName)
```

Comment:
_(Minor)_ I suggest removing this (both overloads) into `FindMethod`.

Reply:
Not clear what do you mean here.